tests/test_loaders.py

```diff
@@ -278,6 +291,42 @@ def test_track():
             dataset.track("~faketrackid~?!")
 
 
+def test_track_placeholder_case():
```

Comment:
Maybe add explanation, from your own words XD:

This function tests the case where there is no data in data_home. It makes sure that the track can be initialized and the attributes accessed, but that anything requiring data files errors (all properties and cached properties).